src/app/stationform/stationform.component.html

```diff
@@ -0,0 +1,14 @@
+<h1>Station hinzufügen</h1>
+<p>
+	<label>Stations Name:</label>
+	<input type="text" [(ngModel)]="newGeoJson.stationName" id="stationname">
+</p>
+<p>
+	<label>Breitenangabe:</label>
+	<input type="number" [(ngModel)]="newGeoJson.lat" id="latitude">
+</p>
+<p>
+	<label>Höhenangabe:</label>
+	<input type="number" [(ngModel)]="newGeoJson.lon" id="longitude">
+</p>
+<button type="button"  (click) = "geoJsonData.pushGeoJsonElement(newGeoJson)" >Speichern</button>
```

Comment:
please no spaces inside a tag.
```suggestion
<button type="button"  (click)="geoJsonData.pushGeoJsonElement(newGeoJson)">Speichern</button>
```